README.md

```diff
@@ -9,3 +9,5 @@ decimal64.
 
 The goal is to fully support decimal64 and decimal128. Currently only supports
 decimal64, excluding subnormals, and needs more thorough testing.
+
+[To gain more familiarity with this project you can refer to new starter guide.](docs/newStarters.md)
```

Comment:
Only "new starter guide" should be linkified. Also, it should be clear that this is for people wanting to contribute to the project, not all users.